packages/components/addon/components/hds/button-set/index.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+// import { assert } from '@ember/debug';
```

Comment:
I can delete this commented out line if I don't need to add any asserts here.

Is the below enough or are there additional items I should add?